distributionviewer/core/static/css/fetching.styl

```diff
@@ -0,0 +1,45 @@
+@import 'distributionviewer';
```

Comment:
It looks like this imports all of the rules from *distributionviewer.styl* into this file. The compressor ultimately gets rid of duplicates, but to be safe we could consider putting variables and other re-usable code into other files and importing them as needed. That's what we did on MDN at least, but I'm open to suggestions.